setup.cfg

```diff
@@ -43,6 +43,7 @@ python_requires = >=3.8
 include_package_data = True
 install_requires =
     appdirs>=1.4.4
+    app-model
     cachey>=0.2.1
```

Comment:
probably good to add `in-n-out` too even if `app-model` dependency as I think we explicitly import it now too